addons/sale_project/models/project.py

```diff
@@ -259,6 +259,7 @@ def get_panel_data(self):
         return {
             **panel_data,
             'sale_items': self._get_sale_items(),
+            'allow_billable': self.allow_billable,
```

Comment:
I think we could avoid adding `sale_items` key is the project is not billable or the user has not the `analytic.group_analytic_accounting` group instead of doing those checks in the JS code. Because here, the server gives the data to the frontend and the frontend does not display this data because the condition is falsy. So why not avoiding sending that data if we are sure the user cannot see that?